great_expectations/rule_based_profiler/data_assistant/data_assistant.py

```diff
@@ -89,11 +89,8 @@ def __new__(cls, clsname, bases, attrs):
                 DataAssistantDispatcher,
             )
 
-            if newclass.__alias__:
```

Comment:
@TrangPham I am getting an error in PyCharm that `newclass.__alias__` does not exist.  I thin that this is because it is defined only in the subclass.  Hence, reverting for now.  Thank you.

Reply:
Hi @alexsherstinsky is type-checking error? Would it help to do a #noqa-type escape here?

Reply:
I think it's probably a type-checking error. My VScode does not complain and all tests were passing.